containerized_ml_models/api/model.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+import pickle as pkl
+import numpy as np
+
+
+class Model:
+
+    __create_key = object()
+    class_instance = None
+
+    def __init__(self, model_path: str, create_key: object):
```

Comment:
Missing return type. It's always None but keep it consisten with other declarations

Reply:
Fixed